vue/templates/_package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "bs-todo",
+  "version": "6.0.1",
+  "description": "A decentralized Todo app built on Blockstack",
+  "author": "Michael Stivala <michaelstivala@me.com> (https://michaelstivala.com/)",
+  "contributors": [
+    "Aaron Blankstein <aaron@blockstack.com> (https://github.com/kantai)",
+    "Larry Salibra <hello@larrysalibra.com> (https://larrysalibra.com/)",
+    "Arjun Balaji <arjunblj@gmail.com> (https://github.com/arjunblj)",
+    "Kim Hallberg <work@hallberg.kim> (https://hallberg.kim/)",
+    "Guy Lepage <guylepage3@gmail.com> (https://lepage.cc/)",
+    "Jack Zampolin (https://github.com/jackzampolin)",
+    "Moxiegirl (https://github.com/moxiegirl)"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/blockstack/blockstack-todos"
+  },
+  "private": true,
+  "scripts": {
+    "start": "vue-cli-service serve",
+    "build": "vue-cli-service build",
+    "lint": "vue-cli-service lint",
+    "test:unit": "vue-cli-service test:unit --coverage --coverageReporters=lcov",
```

Comment:
When running `npm run test:unit` I'm getting an error:
```
> vue-cli-service test:unit --coverage --coverageReporters=lcov

Error: Could not resolve a module for a custom reporter.
  Module name: jest-junit
    at options.reporters.reporters.map.reporterConfig (/Users/matt/Projects/blockstack-app-generator/.vue-gen-test/node_modules/jest-config/build/normalize.js:419:15)
    at Array.map (<anonymous>)
    at normalizeReporters (/Users/matt/Projects/blockstack-app-generator/.vue-gen-test/node_modules/jest-config/build/normalize.js:397:33)
    at normalize (/Users/matt/Projects/blockstack-app-generator/.vue-gen-test/node_modules/jest-config/build/normalize.js:494:5)
    at readConfig (/Users/matt/Projects/blockstack-app-generator/.vue-gen-test/node_modules/jest-config/build/index.js:184:68)
    at getConfigs (/Users/matt/Projects/blockstack-app-generator/.vue-gen-test/node_modules/jest/node_modules/jest-cli/build/cli/index.js:473:77)
    at /Users/matt/Projects/blockstack-app-generator/.vue-gen-test/node_modules/jest/node_modules/jest-cli/build/cli/index.js:251:23
    at Generator.next (<anonymous>)
    at step (/Users/matt/Projects/blockstack-app-generator/.vue-gen-test/node_modules/jest/node_modules/jest-cli/build/cli/index.js:206:30)
    at /Users/matt/Projects/blockstack-app-generator/.vue-gen-test/node_modules/jest/node_modules/jest-cli/build/cli/index.js:225:14
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! bs-todo@6.0.1 test:unit: `vue-cli-service test:unit --coverage --coverageReporters=lcov`
npm ERR! Exit status 1
```

I'm not really familiar with Vue or what code/tests this is trying to exercise -- should we fix this or remove it?